### PR TITLE
User 1 PULL REQUEST HW1

### PR DESCRIPTION
**Pull request by dima666Sik**
ZD 1-3

### CODE REVIEW

File: home_work_1_zd_1.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        for (let j = i; j < mass.length; j++) {
+            if (temp_min > mass[j]) {
+                temp_min = mass[j];
+            }
+            else if (temp_max < mass[j]) {
+                temp_max = mass[j];
+            }
+        }
+    }
+    obj = { min: temp_min, max: temp_max };
```
Comment (StanislawVictorovich) [inline home_work_1_zd_1.js]: Remove please obj variable and make directly return ( return { min:..., max: .... } )

File: home_work_1_zd_1.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+const MASSNUMS = [4, 7, 9, 2, 8, 4, 10, 15, 6, 19];
+let obj = {};
+
+function get_min_and_max(mass) {
```
Comment (StanislawVictorovich) [inline home_work_1_zd_1.js]: please follow documented naming for the JS functions

File: home_work_1_zd_2.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+const MASSNUMS = [4, 7, 9, 2, 8, 4, 10, 15, 6, 19];
+let obj = {};
+
+function get_avg(mass) {
```
Comment (StanislawVictorovich) [inline home_work_1_zd_2.js]: please follow documented naming for the JS functions

File: home_work_1_zd_3.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
+const MASSNUMS = [4, 7, 9, 2, 8, 4, 10, 15, 6, 19];
+// const MASSNUMS = [-1, 0, 1, 2, 4, 5];
+let flag = true;
+
+function get_avg(mass) {
```
Comment (StanislawVictorovich) [inline home_work_1_zd_3.js]: please follow documented naming for the JS functions

File: home_work_1_zd_3.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
+const MASSNUMS = [4, 7, 9, 2, 8, 4, 10, 15, 6, 19];
+// const MASSNUMS = [-1, 0, 1, 2, 4, 5];
+let flag = true;
```
Comment (StanislawVictorovich) [inline home_work_1_zd_3.js]: move flag inside the function scope

File: home_work_1_zd_1.js (hunk 1 above)
Comment (dima666Sik) [reply home_work_1_zd_1.js]: Done

File: home_work_1_zd_1.js (hunk 2 above)
Comment (dima666Sik) [reply home_work_1_zd_1.js]: Done

File: home_work_1_zd_2.js (hunk 3 above)
Comment (dima666Sik) [reply home_work_1_zd_2.js]: Done

File: home_work_1_zd_3.js (hunk 4 above)
Comment (dima666Sik) [reply home_work_1_zd_3.js]: Done

File: home_work_1_zd_3.js (hunk 5 above)
Comment (dima666Sik) [reply home_work_1_zd_3.js]: Done

**Pull request merged**